### ISSUE TITLE
Swift 5.8 regression: `Bundle` is no longer `Sendable`

### ISSUE THREAD

**Issue by NachoSoto** [apple/swift#63763]
**Description**

`Bundle` was `Sendable` in Swift 5.7.x. Starting with Xcode 14.3 beta 1 and Swift 5.8 I see that that extension has been marked `unavailable`:
```swift
@available(*, unavailable)
extension Bundle : @unchecked Sendable {
}
```

Was that intended?

**Environment**
- Swift: `swift-driver version: 1.75.1 Apple Swift version 5.8 (swiftlang-5.8.0.117.11 clang-1403.0.22.8.60)`
- Xcode: `14.3 (14E5197f)`
- Deployment target: iOS 12.0

**Comment by AnthonyLatsis** [comment]
@parkera

**Comment by parkera** [comment]
Yes, it needs a change to be safely Sendable. We can track that with this if you like. We are tracking it internally as well.

**Comment by parkera** [comment]
Unfortunately this correction was necessary for these releases to reflect the reality of how `Bundle` has always behaved, until we have an opportunity to fix the locking inside the type itself. I don't anticipate it changing for the Xcode 14.3 SDK release.

**Comment by NachoSoto** [comment]
Ok thanks for letting us know 🙏🏻

**Comment by AnthonyLatsis** [comment]
@hamishknight Could you transfer this too?
